openedx/core/djangoapps/user_authn/views/logout.py

```diff
@@ -59,7 +60,12 @@ def dispatch(self, request, *args, **kwargs):
 
         # If we are using studio logout directly and there is not OIDC logouts we can just redirect the user
         if settings.FEATURES.get('DISABLE_STUDIO_SSO_OVER_LMS', False) and not self.oauth_client_ids:
```

Comment:
What if we try to reduce "Cyclomatic Complexity" using just one if else expression:
```suggestion
if settings.FEATURES.get('DISABLE_STUDIO_SSO_OVER_LMS', False) and not self.oauth_client_ids and not configuration_helpers.get_value('ALLOW_SAML_LOGOUT_FROM_URL', False):
    response = redirect(self.target)
else:
    response = super(LogoutView, self).dispatch(request, *args, **kwargs)

``

Reply:
Yes, I thought it as you suggested, but I don't know if a three-line condition is more complex than a separated if condition, so let me know what you think @diegomillan

Reply:
I think it's better one if-else clause than nested clauses. @Squirrel18 regardless of the condition length.

Reply:
Addresed. @diegomillan